### ISSUE TITLE
Investigate Crostini support

### ISSUE THREAD

**Issue by bketelsen** [bketelsen/lxdev#21]
Crostini in ChromeOS uses LXC/D under the covers to provide a linux VM.  Can lxdev be tweaked (or forked?) to assist with that?

**Comment by markstos** [comment]
I don't think so. The outer Chrome OS is immutable. Inside that runs a custom VM that primarily just runs LXD. I think this is where `devlx` might be installed to be useful, but it's immutable, too, although you can still log into it and run commands.

An alternate interesting target might be Fedora Silverblue. It also features a minimal, immutable host OS with a focus on running apps in containers. The default tooling is for OCI containers, but `lxd` is easily installed via `snap`.   The Silverblue host layout is a bit different than convention because only `/var/` and `/etc/` are immutable on the host, so `/home` is actually `/var/home` among other changes.
